web-client/iron-remote-gui/src/services/PublicAPI.ts

```diff
@@ -23,6 +23,7 @@ export class PublicAPI {
         authToken: string,
         desktopSize?: DesktopSize,
         preConnectionBlob?: string,
+        kdc_proxy_url?: string,
```

Comment:
You should update the README.md for iron-remote-gui

Reply:
updated